### ISSUE TITLE
PF 11.1 on Debian 11 - Logs full of dbus notice "Rejected: destination has a full message queue"

### ISSUE THREAD

**Issue by heapdavid** [inverse-inc/packetfence#7139]
**Describe the bug**

We are seeing lots of the following in the logs on a PF 11.1 system we've just set up:

```
[notice] [system] Rejected: destination has a full message queue, 0 matched rules; type="signal", sender=":1.0" (uid=0 pid=1 comm="/sbin/init ") interface="org.freedesktop.DBus.Properties" member="PropertiesChanged" error name="(unset)" requested_reply="0" destination="org.freedesktop.DBus" (uid=0 pid=1517 comm="/usr/local/fingerbank/collector/fingerbank-collect")
```

Looks to be coming from fingerbank collector.

**To Reproduce**
1. Install PF11.1 on Debian Bullseye
2. Check the dbus-daemon logs

**Comment by julsemaan** [comment]
Is this a cluster or standalone server?

**Comment by heapdavid** [comment]
It's standalone

**Comment by julsemaan** [comment]
Are you seeing a high CPU or RAM usage of the collector when that happens. It's hard to tell why this is happening as the fingerbank-collector doesn't interact directly with dbus

**Comment by heapdavid** [comment]
No, we're not seeing high resource usage of any kind, and the server has not been put into service yet so has no clients connected to it. It does seem to also be present on our other PF 11.1 servers 

We see about 3-5 lines of this per second in the logs.

**Comment by ohmer1** [comment]
Same problem here on PF 13.2. Did you found a solution?
